src/main/scala/com/microsoft/hyperspace/actions/CreateActionBase.scala

```diff
@@ -181,8 +182,19 @@ private[actions] abstract class CreateActionBase(dataManager: IndexDataManager)
       val missingPartitionColumns = getPartitionColumns(df).filter(
         ResolverUtils.resolve(spark, _, columnsFromIndexConfig).isEmpty)
       val allIndexColumns = columnsFromIndexConfig ++ missingPartitionColumns
+
+      // File path value in DATA_FILE_NAME_COLUMN column is stored as String. We normalize
+      // path values in this column by removing extra preceding `/` characters in the path
+      // and store it the same way paths are stored in Content in an IndexLogEntry instance.
+      // Normalizing path values in DATA_FILE_NAME_COLUMN column keeps path representation
+      // unified in Hyperspace (between index lineage and index metadata) and helps performance
+      // by avoiding the need to fix paths (i.e. removing extra `/` characters) each time
+      // value of DATA_FILE_NAME_COLUMN column is read.
```

Comment:
Can you add an example in the comment to explain this comment so there's no confusion?

Reply:
I added an example there. Plz let me know if it is as expected.